apps/transport/lib/transport_web/controllers/conversion_controller.ex

```diff
@@ -0,0 +1,51 @@
+defmodule TransportWeb.ConversionController do
+  use TransportWeb, :controller
+
+  def get(%Plug.Conn{} = conn, %{"resource_id" => resource_id, "convert_to" => convert_to}) do
+    if convert_to in Ecto.Enum.dump_values(DB.DataConversion, :convert_to) do
+      case DB.Resource.get_related_conversion_info(resource_id, convert_to) do
+        nil ->
+          conn |> conversion_not_found()
+
+        %{url: url, resource_history_last_up_to_date_at: %DateTime{} = last_up_to_date_at} ->
+          trace_request(resource_id, convert_to)
+
+          conn
+          |> put_resp_header("etag", md5_hash(url))
+          |> put_resp_header("cache-control", "public, max-age=300")
+          |> put_resp_header("x-last-up-to-date-at", last_up_to_date_at |> DateTime.to_iso8601())
```

Comment:
Tu me diras ce que tu penses des headers ajoutés, ça pourrait être utile pour diminuer notre trafic et indiquer la durée de vie attendue. 300s c'est très peu, c'est en réalité bien plus long la durée de validité d'une conversion mais très variable d'une ressource à l'autre.

Reply:
Salut @AntoineAugusti, la ressource convertie n'est pas disponible tout le temps ?

Reply:
@agougoua Si si, j'ajoute juste des headers pour indiquer que ça peut être gardé en cache "au moins 5mn", vu le rythme de mise à jour des GTFS. Et le `x-last-up-to-date-at` donne une indication de quand la dernière historisation du GTFS date, `etag` donne un hash de la conversion qui est donné.

Le PAN détecte les nouvelles versions des GTFS au max toutes les 6h et les conversions sont effectuées dans la foulée

Reply:
Super !